Code/Editor/EditorViewportWidget.cpp

```diff
@@ -2508,11 +2233,11 @@ void EditorViewportWidget::CenterOnAABB(const AABB& aabb)
     Matrix34 newTM = Matrix34(rotationMatrix, newPosition);
 
     // Set new orbit distance
-    m_orbitDistance = distanceToTarget;
-    m_orbitDistance = fabs(m_orbitDistance);
+    float orbitDistance = distanceToTarget;
+    orbitDistance = fabs(orbitDistance);
 
     SetViewTM(newTM);
-    SandboxEditor::OrbitCameraControlsBus::Event(GetViewportId(), &SandboxEditor::OrbitCameraControlsBus::Events::SetOrbitDistance, m_orbitDistance);
+    SandboxEditor::OrbitCameraControlsBus::Event(GetViewportId(), &SandboxEditor::OrbitCameraControlsBus::Events::SetOrbitDistance, orbitDistance);
```

Comment:
aside: I think this whole function can actually be deleted pretty soon too, if you look in `SandboxIntegrationManager::GoToEntitiesInViewports` we now use a new code path for the new Modular Camera (see `    if (SandboxEditor::UsingNewCameraSystem())` line 1706)